pstress/pstress-run.sh

```diff
@@ -1574,8 +1592,15 @@ pquery_test(){
         sleep 2
       fi
     fi
-    echoit "Killing mysqld server..."
-    (sleep 0.2; kill -9 ${MPID} >/dev/null 2>&1; timeout -k5 -s9 5s wait ${MPID} >/dev/null 2>&1) &  # Terminate mysqld
+
+    if [ $SIGNAL -eq 9 ]; then
+      kill_server 9
+    elif [ $SIGNAL -eq 4 ]; then
+      kill_server 4
+    elif [ $SIGNAL -eq 15 ]; then
```

Comment:
signal 15 also we have to pass to kill. Don't call mysqladmin.

Reply:
Done

Reply:
just do kill_server $SIGNAL

Reply:
Fixed

Reply:
no need to do use if-else loop. 
kill_server $SIGNAL

Reply:
Fixed

Reply:
I can still see this code. 

if [ $SIGNAL -eq 9 ]; then
       kill_server $SIGNAL
     elif [ $SIGNAL -eq 4 ]; then
       kill_server $SIGNAL
     elif [ $SIGNAL -eq 15 ]; then

remove and just used kill_server $SIGNAL